src/runtime_ccall.cpp

```diff
@@ -146,21 +146,27 @@ void *jl_get_library(const char *f_lib)
 #endif
     if (f_lib == NULL)
         return jl_RTLD_DEFAULT_handle;
-    hnd = libMap[f_lib];
+    JL_LOCK_NOGC(&libmap_lock);
```

Comment:
But we are not using `uv_thread_self` https://github.com/JuliaLang/julia/blob/b80dad40ab36bcd7e11859d36b51c6afcf008148/src/julia_threads.h#L136